tests/trybuild_tests_src/contiguous_derive_failure.rs

```diff
@@ -0,0 +1,17 @@
+use bytemuck::Contiguous;
```

Comment:
Just now realizing that these tests won't fail unless the derive macro fails _all_ cases. It might be worth it to only run these tests in CI on stable and keep the stderr files up to date.